File: Assignment44/program1.c
#include<stdio.h>
#include<fcntl.h>
#include<unistd.h>

void OpenFile(char *FileName){
    int fd = 0;

    fd = open(FileName,O_RDWR);

    if(fd == -1){
        printf("Unable to open file\n");
    }
    else{
        printf("File opened successfully with fd : %d\n",fd);

        close(fd);
    }
}

int main(){
    char fName[30] = {'\0'};

    printf("Enter file name to open : \n");
    scanf(" %s",fName);

    OpenFile(fName);

    return 0;
}

File: Assignment45/program1.java
import java.io.File;
import java.util.*;

class OpenFile{
    public static void main(String args[]){
        Scanner sobj = new Scanner(System.in);
        
        System.out.println("Enter file name : ");
        String Filename = sobj.nextLine();

        File fobj = new File(Filename);
        
        if(fobj.exists()){
            System.out.println("File opened successfully.");
        }
        else{
            System.out.println("File does not exist.");
        }

        sobj.close();

    }
}